### ISSUE TITLE
date picker doesn't block off month/year selection when setting a min and/or max date

### ISSUE THREAD

**Issue by jonathan-rvezy** [nathanreyes/v-calendar#1339]
It works fine for the calendar, but not the date picker.

```
      <v-date-picker v-model.range.string="model" :min-date="minDate" :max-date="maxDate" />
      <v-calendar :min-date="minDate" :max-date="maxDate" />
```

<img width="375" alt="image" src="https://github.com/nathanreyes/v-calendar/assets/94866892/b75ea842-4fb7-4b86-8a29-267afd8d7b0c">

**Comment by houfeng0923** [comment]
yes. it lose props: min-date, max-date to calendar from picker.
Now, An alternative temp solution is：pass min-page, max-page explicitly

**Comment by hobuttt** [comment]
Same problem in 3.0.3 (

**Comment by nathanreyes** [comment]
This should be fixed in [3.1.0](https://github.com/nathanreyes/v-calendar/releases/tag/v3.1.0)
